APMrover2/sensors.cpp

```diff
@@ -151,6 +151,17 @@ void Rover::compass_cal_update() {
         compass.compass_cal_update();
     }
 }
+// Save compass offsets
+void Rover::compass_save(){
```

Comment:
This doesn't seem to called from anywhere, I think you missed adding the scheduler entry to Rover: https://github.com/ArduPilot/ardupilot/blob/fba185cefbb7e66c26c25c813796a1b8663e0646/ArduPlane/ArduPlane.cpp#L73